db/seeds/03_rules.rb

```diff
@@ -131,6 +131,18 @@ def rule_id(rule_results, object)
                predicate: 'reason',
                object: 'States Attorney General Filed Suit for Deceptive Action',
                priority: 2),
+      Rule.new(rule_name: CautionFlag.name,
+               matcher: Rule::MATCHERS[:has],
+               subject: nil,
+               predicate: 'reason',
+               object: 'The Federal Trade Commission (FTC) has reached a settlement with this school',
+               priority: 2),
+      Rule.new(rule_name: CautionFlag.name,
```

Comment:
These two rules look identical to me.  Is this intentional?

Reply:
one object field has a period on the end the other doesn't

Reply:
I see that now.  I'm curious what the purpose of that is?